setup.cfg

```diff
@@ -40,6 +39,9 @@ install_requires =
     tabulate
     ruamel.yaml
 
+[options.package_data]
+* = *.jinja2, *.css
```

Comment:
Without this change, installing from archive misses the package data files.

Reply:
@gro1m I'm less familiar with this part, any concerns?

Reply:
@oev81 what did you observe? What do you mean by "installing from archive"? Do you mean from a zip file instead of `pip install pycobertura`? What was your process to get an archive?

Reply:
> @gro1m I'm less familiar with this part, any concerns?

This is about data file support.
In this case all `jinja2` and `css` files are included as well and not only the Python files. 
Concretely, this means when installing pycobertura, we have also access to the following files in the `templates` folder:
- `html.jinja2`
- `html-delta.jinja2`
- `normalize.css`

As we have this under control, this should not be a security issue, but rather an improvement. Probably some html rendering might not have worked before in all cases, as for `include_package_date=True` to have worked properly, an additional `MANIFEST.in` file (see also the reference link below) would have been required.

Data File Support Reference (even if for `setup.py` instead of `setup.cfg`, but this analogue): https://setuptools.pypa.io/en/latest/userguide/datafiles.html

Hope this is clearer now.

Reply:
@aconrad 
You can install package directly from github as an archive
```
pip install https://github.com/oev81/pycobertura/archive/coverage_parsing_speedup.zip
```
or put
```
https://github.com/oev81/pycobertura/archive/coverage_parsing_speedup.zip
```
in requirements.txt

This way is very helpful for me.  https://stackoverflow.com/a/24811490